requirements/doc-requirements.txt

```diff
@@ -16,3 +16,4 @@ torchvision>=0.12.0
 lightning>=1.8.1
 scrapy
 ipywidgets>=8.1.1
+incremental<24.7.0
```

Comment:
`incremental` released a new version today:

https://pypi.org/project/incremental/24.7.0/

Reply:
When can we unpin this?

Reply:
When we upgrade python to 3.9